E.Deezer/Api/User.cs

```diff
@@ -8,13 +8,29 @@ namespace E.Deezer.Api
 {
 	public interface IUser : IObjectWithImage
 	{
-		ulong Id { get;  }
-		string Name { get;  }
-		string Link { get;  }
-		string Country { get;  }
-
-
-		// ** Methods **
+        ulong id { get; set; }
```

Comment:
Could we have properties starting with a captial letter please? It'll match the rest of the API that way.